src/OSPSuite.Infrastructure.Import/Core/DataFormat/AbstractColumnsDataFormat.cs

```diff
@@ -322,6 +332,15 @@ private SimulationPoint parseMappingOnSameColumn(MappingDataFormatParameter curr
 
       private SimulationPoint parseMappingOnSameGivenColumn(MappingDataFormatParameter currentParameter, DataSheet dataSheet, UnformattedRow row)
       {
+         if (!currentParameter.MappedColumn.LloqColumn.IsNullOrEmpty())
+         {
+            ColumnDescription lloqColumnDescription = null;
+            if (!string.IsNullOrEmpty(currentParameter.MappedColumn.LloqColumn))
```

Comment:
I don't understand. Why are checking this if you check it already on line 335?
Can we comment as to WHAT we are doing here also. What is the use case. the function is called `parseMappingOnSameGivenColumn` but we are dealing with LLOQ.

Reply:
and alco create a variable for currentParameter.MappedColumn.LloqColumn
This is used 4 times in 5 lines

Reply:
OK, so double check removed, added a comment and refactor a bit to make it make more sense (and also avoid some code duplication)